aeron-client/src/main/c/util/aeron_bitutil.h

```diff
@@ -89,7 +88,7 @@ inline int aeron_number_of_trailing_zeroes_u64(uint64_t value)
     }
 
     return __builtin_ctzll(value);
-#elif defined(_MSC_VER)
+#elif defined(_MSC_VER) && defined(AERON_CPU_X64)
```

Comment:
Why add the x86 check back?

Reply:
_BitScanForward64 is only available on X64 target